docs/userGuide/components/pic.md

```diff
@@ -23,7 +23,7 @@
 ## Pic Options
 Name | Type | Default | Description 
 --- | --- | --- | ---
-alt | `string` | | The alternative text of the image. Must be specified.
+alt | `string` | | **This must be specified.**<br>The alternative text of the image.

```

Comment:
I was quite surprised to see that `alt` is a compulsory attribute. Is it really true?

Reply:
[Yes](https://www.w3schools.com/tags/att_img_alt.asp) 😆